pkg/kubeapiserver/options/authentication_test.go

```diff
@@ -400,7 +422,343 @@ func TestBuiltInAuthenticationOptionsAddFlags(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if !opts.OIDC.areFlagsConfigured() {
+		t.Fatal("OIDC flags should be configured")
+	}
+	// nil these out because you cannot compare pointers.
```

Comment:
```suggestion
	// nil these out because you cannot compare non-nil functions.
```

(pointers are comparable, go checks that they point to the same memory address)